builds/fedmir

```diff
@@ -0,0 +1,83 @@
+#
+# FedMir - Fedora Package Mirror
+#
+# This container builds an HTTP Package Mirror for fedora package repositories.
+# It builds a package repository with the selected set of packages, the
+# specified target release version, as well as the specified architecture. It
+# then uses the official `httpd` container to serve this repository via HTTP.
+#
+# Arguments:
+#
+#   * FEDMIR_ARCH="x86_64"
+#       This controls the architecture of the packages to fetch.
+#
+#   * FEDMIR_BUILD="docker.io/library/fedora:latest"
+#       This controls the build-container used to fetch fedora packages and
+#       setup the package repository. By default, it uses the latest stable
+#       release of the official fedora containers.
+#
+#   * FEDMIR_HOST="docker.io/library/httpd:2.4"
+#       This controls the host-container used to serve the fedora package
+#       mirror.
+#
+#   * FEDMIR_PACKAGES="base emacs vim"
+#       Specify the packages to include in the mirror. Separate packages by
+#       space. All their dependencies will be included in the mirror.
+#
+#   * FEDMIR_RELEASE="rawhide"
+#       This controls the Fedora release used as base for this mirror. The
+#       packages will be fetched from the official Fedora repositories of the
+#       specified release.
+#
+
+# Import container arguments (must be before any `FROM`).
+ARG     FEDMIR_BUILD="docker.io/library/fedora:31"
+ARG     FEDMIR_HOST="docker.io/library/httpd:2.4"
+
+# Fetch our build environment.
+FROM    "${FEDMIR_BUILD}" AS build
+
+# Import our mirror parameters.
+ARG     FEDMIR_ARCH
+ARG     FEDMIR_PACKAGES
+ARG     FEDMIR_RELEASE
```

Comment:
These are documented as having default values `"x86_64"`, `"base emacs vim"`, and `"rawhide"`, but the default is missing here.

Note that `base` is not a valid package name. There's `basesystem`. Maybe we could just use whatever is in the `@core` comps group? (However, `dnf download` doesn't support downloading groups. I have a patch upcoming for that.)